pkg/eval/builtin_fn_misc_test.go

```diff
@@ -106,10 +106,11 @@ func TestSleep(t *testing.T) {
 		That(`sleep -7`).Throws(ErrNegativeSleepDuration, "sleep -7"),
 		That(`sleep -3h`).Throws(ErrNegativeSleepDuration, "sleep -3h"),
 
-		// Verify the correct behavior if a number rather than a string is
-		// passed to the command.
-		//
-		// TODO: Augment these tests to use `(num ...)` as part of resolving https://b.elv.sh/1317.
+		That(`sleep 33/3`).Puts(11*time.Second), // rational number string
```

Comment:
33/3 is internally normalized to the same as 11. To test a real rational number, use 1/2